src/ir/cost.h

```diff
@@ -726,6 +726,10 @@ struct CostAnalyzer : public OverriddenVisitor<CostAnalyzer, CostType> {
     return 8 + visit(curr->ref) + visit(curr->num);
   }
 
+  CostType visitContNew(ContNew* curr) {
+    // Some arbitrary "high" value, reflecting that this may allocate a stack
+    return 10 + visit(curr->func);
```

Comment:
Would you expect that this would be cheaper than `resume`, which we model with a base cost of 12?

Reply:
Ha, good point! If the stack allocation does happen at `cont.new`, this should indeed be more expensive than `resume`. But I could also imagine an implementation strategy where the allocation happens lazily when `resume`-ing a new continuation for the first time.
In general, I have little confidence in the numbers here. For now, I've increased the base cost for `cont.new` to 14, which is still very arbitrary, but at least greater than the value for `resume`. Should I add a TODO that we should probably revisit these values at some point? (e.g., once we have more experience from implementing the extension)

Reply:
Ehh, probably no need for a TODO. All of the numbers in this file are similarly suspect.